### PR TITLE
note formatting and captions

### PR DESCRIPTION
**Pull request by emilybrown1**
Part of a quality review of HDRP docs.

Fixed note formatting and moved image captions.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Main.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+The High Definition Render Pipeline (HDRP) includes its own purpose-built implementation for [post-processing](https://docs.unity3d.com/Manual/PostProcessingOverview.html). This is built into HDRP, so you don't need to install any other package.
 
-This implementation uses the [Volume](Volumes.md) system. You add post-processing effects to your Camera in the same way you add any other [Volume Override](Volume-Components.md). Note that HDRP already enables some post-processing effects in the [HDRP Global Settings](Default-Settings-Window.md), go to Section **Volume Profiles**.
+This implementation uses the [Volume](Volumes.md) system. You add post-processing effects to your Camera in the same way you add any other [Volume Override](Volume-Components.md). 
 
-The images below show a Scene with and without HDRP post-processing.
+**Note**: HDRP already enables some post-processing effects in the [HDRP Global Settings](Default-Settings-Window.md), go to Section **Volume Profiles**.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Main.md:7]: If you mean the Volume profile section in this doc, link to it. 
To do that, use a header link and then a md link with the format [Header title](#header-title).
(I can find an example of a header link if you need one.)

Comment (Vic-Cooper) [reply com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Main.md:7]: Also I'd change the phrasing here to say some things are enabled by default (I know it sounds like passive voice but checkboxes are the exception): 

"Some post-processing effects are enabled by default in the [HDRP Global Settings](Default-Settings-Window.md)..."

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
small notes.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
As discussed :) header link etc...

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**